compiler/circle-opselector-test/value_test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script compares the selected circle model by recipe and by 'circle-opselector'
+# And to compare the two circle models, 'luci-value-test' which compare equivalence of tflite and circle model was used.
+#
+# HOW TO USE
+#
+# ./value_test.sh <path/to/bin_dir> <path/to/work_dir> <path/to/venv_dir> <path/to/circle-opselector-dir> <TEST 1> <TEST 2> ...
+LUCI_VERIFY_SOURCE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../luci-value-test/ && pwd)"
+LUCI_VERIFY_SCRIPT_PATH="${LUCI_VERIFY_SOURCE_PATH}/luci_eval_verifier.py"
```

Comment:
Please don't use `../luci-value-test`, this will make this module depend on `luci-value-test`.
You can make a copy of that script in this module.

Reply:
I will do that